### PR TITLE
fix: After selecting multiple directories and right clicking to open the vault, the breadcrumbs display the complete path of the vault

### PR DESCRIPTION
**Pull request by liyigang1**
When modifying and opening, check if it is a directory or if it is an internal jump in the document management system

Log: After selecting multiple directories and right clicking to open the vault, the breadcrumbs display the complete path of the vault
Bug: https://pms.uniontech.com/bug-view-214937.html

### CODE REVIEW

File: src/plugins/common/core/dfmplugin-menu/menuscene/fileoperatormenuscene.cpp [hunk 1]
```diff
@@ -207,6 +207,19 @@ bool FileOperatorMenuScene::triggered(QAction *action)
                 dpfSignalDispatcher->publish(GlobalEventType::kChangeCurrentUrl, d->windowId, cdUrl);
             }
         } else {
+            for ( auto it = d->selectFiles.begin(); it != d->selectFiles.end();) {
+                auto info = InfoFactory::create<FileInfo>(*it);
+                if (info.isNull() || !info->isAttributes(OptInfoType::kIsDir) ) {
```
Comment (max-lvs) [inline src/plugins/common/core/dfmplugin-menu/menuscene/fileoperatormenuscene.cpp:214]: 备注一下，打开文件夹的特殊处理

Comment (liyigang1) [reply src/plugins/common/core/dfmplugin-menu/menuscene/fileoperatormenuscene.cpp:214]: 已备注

**Review by max-lvs — APPROVED** [review]

**Pull request merged**